app/assets/javascripts/map.js

```diff
@@ -34,6 +44,9 @@ $(function() {
       url: "/api/courts",
     });
     courtsRequest.done(plotCourts);
+    map.on("click", function(){
```

Comment:
Instead of using an anonymous function like this what about extracting another function called `hideInformationBox` to convey your intent.